test/com/facebook/buck/android/testdata/android_instrumentation_test_integration_test/apps/myapp_test/BUCK

```diff
@@ -0,0 +1,14 @@
+android_instrumentation_apk(
+  name = 'app_test',
+  manifest = 'AndroidManifest.xml',
+  apk = '//apps/myapp:app-exo',
+  deps = [
+    '//test/com/example/activity:activity_test',
+  ],
+)
+
+android_instrumentation_test(
+  name = 'test',
+  apk = '//apps/myapp:app-exo',
```

Comment:
Yep that's what I meant. Shouldn't need changes to android_instrumentation_apk, right?